lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,15 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    def initialize(attrs = {})
+      @raw = attrs
+    end
+
+    def values
```

Comment:
@gtimti `values` and `headers` attribute would not just work out of the box since response does not have `attributes` node/key. However, your point led me to a thought that if I change the way how I override `initialize` from keeping `@raw` to keeping `@attributes` instead, then using attributes will work 😉